AntaresWebLinux.spec

```diff
@@ -1,25 +1,27 @@
 # -*- mode: python ; coding: utf-8 -*-
 from pathlib import Path
-from PyInstaller.utils.hooks import collect_dynamic_libs
 
 block_cipher = None
 
-# We need to analyze all alembic files to be sure the migration phase works fine
-migrations_dir = Path('alembic/versions')
-migration_files = [str(f) for f in migrations_dir.iterdir() if f.is_file() and f.suffix == '.py']
+# We need to analyze all alembic files to be sure the migration phase works fine:
+# alembic loads version files by their path, so we need to add them as "data" to the package,
+# but all the dependencies they use need to be included also, wo we need to perform a
+# dedicated analyse for this.
+versions_dir = Path('alembic/versions')
+versions_files = [str(f) for f in versions_dir.iterdir() if f.is_file() and f.suffix == '.py']
+alembic_analysis = Analysis(["alembic/env.py"] + versions_files)
 
-binaries = [('./alembic.ini', './alembic.ini')]  + collect_dynamic_libs('tables')
-
-antares_web_server_a = Analysis(['antarest/gui.py', 'alembic/env.py'] + migration_files,
+antares_web_server_a = Analysis(['antarest/gui.py'],
              pathex=[],
-             binaries=binaries,
-             datas=[('./resources', './resources'), ('./alembic', './alembic')],
+             binaries=[],
+             datas=[('./resources', './resources'), ('./alembic', './alembic'), ('./alembic.ini', './')],

```

Comment:
Why did you add `./` ?

Reply:
Before we had this: `binaries = [('./alembic.ini', './alembic.ini')]` which created a folder called "alembic.ini" in which the file was placed.

Now the file is correctly placed at the root